README.rst

```diff
@@ -58,12 +58,164 @@ Python:
   provides an abstraction of NI-DAQmx in the ``nidaqmx`` module, which collides
   with this package's module name.
 
+Getting Started
+===============
+
+In order to use the **nidaqmx** package, you must have at least one DAQ device installed on your
+machine. X series DAQ device (eg: PCIe-6363 or USB-6351) is recommended, physical or
+simulated devices are supported. You can use the **NI MAX** or **NI Hardware Configuration Utility**
+to verify and configure your devices.
+
+Verify and configure devices on **NI MAX**:
```

Comment:
"Verify and configure devices" is very general, and I think this image is demonstrating something more specific.